dashboard/app/controllers/pd/workshop_enrollment_controller.rb

```diff
@@ -45,6 +45,8 @@ def create
       render :full
     else
       @enrollment = ::Pd::Enrollment.new workshop: @workshop
+      @enrollment.build_school_info school_info_params
```

Comment:
The downside to build is that I think it's going to add additional instances for same school districts.  Perhaps I can do a lookup first and use an existing one if it matches?